fmt/format.h

```diff
@@ -3446,9 +3537,6 @@ class WindowsError : public SystemError {
      }
    \endrst
   */
-  WindowsError(int error_code, CStringRef message) {
-    init(error_code, message, ArgList());
-  }
```

Comment:
And this

Reply:
This one is also generated by the FMT_VARIADIC_ macro.

Reply:
This should be resolved with the newly added overload.